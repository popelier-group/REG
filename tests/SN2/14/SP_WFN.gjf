# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.85518697   -0.00005160    0.00020515
 H                 -0.88263797   -0.90942860   -0.56659285
 H                 -0.88263397    0.94551440   -0.50390485
 H                 -0.88261997   -0.03624660    1.07115615
 Br                 1.61605503    0.00000440   -0.00001885
 Cl                -3.27735997    0.00001240   -0.00004685

frame.wfn
