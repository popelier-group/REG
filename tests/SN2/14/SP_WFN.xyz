
6

C        0.000082    0.009251    0.741487
H       -0.555272   -0.906747    0.780339
H       -0.515851    0.948690    0.757073
H        1.071398   -0.013162    0.769371
Br        0.000246   -0.021684   -1.729562
Cl       -0.000552    0.039685    3.163468
