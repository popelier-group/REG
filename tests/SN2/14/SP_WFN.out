 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.14/Gau-192289.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.14/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    192291.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.85519  -0.00005   0.00021 
 H                    -0.88264  -0.90943  -0.56659 
 H                    -0.88263   0.94551  -0.5039 
 H                    -0.88262  -0.03625   1.07116 
 Br                    1.61606   0.       -0.00002 
 Cl                   -3.27736   0.00001  -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.855187   -0.000052    0.000205
      2          1           0       -0.882638   -0.909429   -0.566593
      3          1           0       -0.882634    0.945514   -0.503905
      4          1           0       -0.882620   -0.036247    1.071156
      5         35           0        1.616055    0.000004   -0.000019
      6         17           0       -3.277360    0.000012   -0.000047
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.071905   0.000000
     3  H    1.071903   1.856002   0.000000
     4  H    1.071914   1.855982   1.855983   0.000000
     5  Br   2.471242   2.718739   2.718701   2.718843   0.000000
     6  Cl   2.422173   2.623500   2.623463   2.623656   4.893415
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000082    0.009251    0.741487
      2          1           0       -0.555272   -0.906747    0.780339
      3          1           0       -0.515851    0.948690    0.757073
      4          1           0        1.071398   -0.013162    0.769371
      5         35           0        0.000246   -0.021684   -1.729562
      6         17           0       -0.000552    0.039685    3.163468
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         145.5731350           0.8499812           0.8499812
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       172.0615708349 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  6.56D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.38707578     A.U. after   12 cycles
            NFock= 12  Conv=0.31D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.62031-101.31646 -62.27334 -56.09221 -56.09052
 Alpha  occ. eigenvalues --  -56.09052 -10.09078  -9.22965  -8.48222  -6.99043
 Alpha  occ. eigenvalues --   -6.98660  -6.98660  -6.30872  -6.30315  -6.30315
 Alpha  occ. eigenvalues --   -2.42040  -2.41888  -2.41888  -2.41449  -2.41449
 Alpha  occ. eigenvalues --   -0.59966  -0.56540  -0.52403  -0.30883  -0.30882
 Alpha  occ. eigenvalues --   -0.17428  -0.11022  -0.11022  -0.10410  -0.10410
 Alpha  occ. eigenvalues --   -0.09554
 Alpha virt. eigenvalues --    0.09241   0.11491   0.15099   0.15100   0.15849
 Alpha virt. eigenvalues --    0.17783   0.18526   0.18526   0.20358   0.21597
 Alpha virt. eigenvalues --    0.21597   0.23129   0.23129   0.24711   0.30318
 Alpha virt. eigenvalues --    0.31707   0.31708   0.36592   0.39070   0.53718
 Alpha virt. eigenvalues --    0.54673   0.54673   0.57225   0.57225   0.60026
 Alpha virt. eigenvalues --    0.64238   0.64238   0.69708   0.76899   0.76899
 Alpha virt. eigenvalues --    0.76990   0.82755   0.82755   0.83900   0.83901
 Alpha virt. eigenvalues --    0.85138   0.89754   1.09234   1.09235   1.09659
 Alpha virt. eigenvalues --    1.09660   1.12328   1.17226   1.22613   1.22614
 Alpha virt. eigenvalues --    1.33581   1.52505   1.61838   1.61839   1.67471
 Alpha virt. eigenvalues --    1.96013   2.04092   2.04093   2.08993   2.08995
 Alpha virt. eigenvalues --    2.19166   2.19166   2.31078   2.47526   2.47528
 Alpha virt. eigenvalues --    2.56197   2.61687   2.61687   2.68972   2.77793
 Alpha virt. eigenvalues --    2.77793   2.98368   3.22015   3.86811   3.86812
 Alpha virt. eigenvalues --    4.33833   4.33834   4.35990   4.35991   4.54707
 Alpha virt. eigenvalues --    6.75437   7.74650   7.74651   7.82772   9.86490
 Alpha virt. eigenvalues --   24.04155  26.03806  26.03807  26.25807  48.05694
 Alpha virt. eigenvalues --  215.84049 289.95107 289.95107 290.031401020.82618
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.055159   0.419941   0.419946   0.419925   0.041110  -0.043522
     2  H    0.419941   0.513549  -0.032119  -0.032130  -0.033735  -0.018763
     3  H    0.419946  -0.032119   0.513548  -0.032128  -0.033741  -0.018765
     4  H    0.419925  -0.032130  -0.032128   0.513556  -0.033715  -0.018755
     5  Br   0.041110  -0.033735  -0.033741  -0.033715  35.679311   0.010042
     6  Cl  -0.043522  -0.018763  -0.018765  -0.018755   0.010042  17.697695
 Mulliken charges:
               1
     1  C   -0.312558
     2  H    0.183257
     3  H    0.183259
     4  H    0.183248
     5  Br  -0.629273
     6  Cl  -0.607932
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.237206
     5  Br  -0.629273
     6  Cl  -0.607932
 Electronic spatial extent (au):  <R**2>=           1142.3102
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0011    Y=             -0.0488    Z=             -3.8792  Tot=              3.8796
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.8336   YY=            -43.8403   ZZ=            -87.6031
   XY=              0.0001   XZ=              0.0076   YZ=             -0.5491
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             14.5921   YY=             14.5854   ZZ=            -29.1774
   XY=              0.0001   XZ=              0.0076   YZ=             -0.5491
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.1104  YYY=             -0.5398  ZZZ=           -145.3299  XYY=             -1.0994
  XXY=             -0.2736  XXZ=            -16.2069  XZZ=              0.0184  YZZ=             -1.4162
  YYZ=            -16.2241  XYZ=              0.0140
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -71.0688 YYYY=            -71.2821 ZZZZ=          -1760.8043 XXXY=              0.0112
 XXXZ=              0.8958 YYYX=             -0.0280 YYYZ=             -8.5537 ZZZX=              0.1636
 ZZZY=            -12.5895 XXYY=            -23.7271 XXZZ=           -252.3433 YYZZ=           -252.4321
 XXYZ=             -2.9178 YYXZ=             -0.7480 ZZXY=              0.0208
 N-N= 1.720615708349D+02 E-N=-7.696771536131D+03  KE= 3.069967512764D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE764\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.85518697,-0.0000516,0.00020515\H,0,-0.88263797,-0.9094286,-0.56
 659285\H,0,-0.88263397,0.9455144,-0.50390485\H,0,-0.88261997,-0.036246
 6,1.07115615\Br,0,1.61605503,0.0000044,-0.00001885\Cl,0,-3.27735997,0.
 0000124,-0.00004685\\Version=ES64L-G16RevC.01\State=1-A\HF=-3074.38707
 58\RMSD=3.077e-09\Dipole=1.5263348,-0.0000485,0.0001977\Quadrupole=-21
 .6978395,10.8489824,10.8488572,0.0001498,-0.0005379,-0.0000023\PG=C01 
 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 The mind is not a vessel to be filled but a fire to be kindled.
                                  -- Plutarch
 Job cpu time:       0 days  0 hours  0 minutes 16.2 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  9.6 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:46:58 2020.
